### ISSUE TITLE
GitHub repo fails to fetch

### ISSUE THREAD

**Issue by dankohn** [cncf/landscapeapp#72]
From https://app.netlify.com/sites/landscape/deploys/5c2a908526d25a1e7805271b
```sh
5:01:34 PM: > interactive-landscape@1.0.47 stop-ci /opt/build/repo/node_modules/interactive-landscape
5:01:34 PM: > kill -9 `cat /tmp/ci.pid` && rm /tmp/ci.pid
5:01:35 PM: Host key verification failed.
5:01:35 PM: fatal: Could not read from remote repository.
5:01:35 PM: Please make sure you have the correct access rights
5:01:35 PM: and the repository exists.
5:01:35 PM: child_process.js:643
```
From https://www.netlify.com/docs/github-permissions/#accessing-other-repositories-at-build it seems like the issue could be accessing https://github.com/cncf/landscape.git instead of git@github.com:cncf/landscape.git.

See https://stackoverflow.com/questions/13363553/git-error-host-key-verification-failed-when-connecting-to-remote-repository for some additional ideas.

Holding off until we can reliably reproduce.

**Comment by ZeusTheTrueGod** [comment]
Not sure if I can reproduce it.

**Comment by dankohn** [comment]
Happened again: https://app.netlify.com/sites/landscape/deploys/5c2cf9d04e0aaf836c284418

**Comment by lukaszgryglicki** [comment]
Possibly related: GitHub was down just a few minutes ago.

**Comment by dankohn** [comment]
Twice in a row, and GitHub is back up: https://app.netlify.com/sites/landscape/deploys/5c2cfcc578211c8f998c3133

**Comment by dankohn** [comment]
Fixed with 9f59f726fb28316b0ff892627d5e2c83cfb745be
